publ/entry.py

```diff
@@ -196,7 +196,7 @@ def _get_first(query):
     def _previous(self, **kwargs):
         # Get the previous item in any particular category
         spec = {
-            'category': self._record.category,
+            'category': kwargs.get('category', self._record.category),
             'recurse': 'category' in kwargs
```

Comment:
this should probably instead be set to `kwargs['recurse']` with a fallback to true if the requested category is a parent of this entry's (requesting a child category is undefined behavior and should maybe throw an exception)